### PR TITLE
Remove "Show popup" sample

### PR DESCRIPTION
**Pull request by amweis226**
# Description

Removes the "Show popup" sample from the Samples list in the WPF, UWP, and MAUI platforms.

## Type of change

- [x] Sample removal

**Platforms tested on**:

- [x] WPF .NET 6
- [x] UWP
- [x] .NET MAUI

## Checklist

- [x] Runs and compiles on all active platforms
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] Self-review of changes
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [x] `sample_sync.py` runs without making changes

### CODE REVIEW

File: src/ArcGISRuntime.Viewers.All.sln [hunk 1]
```diff
@@ -284,6 +284,7 @@ Global
 	EndGlobalSection
 	GlobalSection(SharedMSBuildProjectFiles) = preSolution
 		ArcGISRuntime.Samples.Shared\ArcGISRuntime.Samples.Shared.projitems*{0ad85845-0ba1-4dc9-8c32-e6fd7632e32d}*SharedItemsImports = 13
+		ArcGISRuntime.Samples.Shared\ArcGISRuntime.Samples.Shared.projitems*{7f7b3255-9ea0-4296-b690-14be9e2204f3}*SharedItemsImports = 5
```
Comment (ZackAllen) [inline src/ArcGISRuntime.Viewers.All.sln:287]: This change is unrelated.

**Review by duffh — APPROVED** [review]

**Review by ZackAllen — APPROVED** [review]
